pdp/context.go

```diff
@@ -70,8 +73,8 @@ type Context struct {
 // name and value. If "f" function returns error NewContext stops iterations
 // and returns the same error. All pairs of attribute name and type should be
 // unique.
-func NewContext(c *LocalContentStorage, count int, f func(i int) (string, AttributeValue, error)) (*Context, error) {
-	ctx := &Context{a: make(map[string]interface{}, count), c: c}
+func NewContext(c *LocalContentStorage, p *ps.ConnectionManager, count int, f func(i int) (string, AttributeValue, error)) (*Context, error) {
```

Comment:
I'd prefer to move connection management to policy storage. All references to pip reside in policy so we can add (similarly to `attributes` section) `pip` section to policy which will contain additional options for pip connections. The section will originate the manager (which I'd prefer to call pool for now).